lib/list_all.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+class ListAll extends StatelessWidget {
+  const ListAll({super.key, required this.listOfTasks, required this.title});
```

Comment:
In the flutter repo, the style guide specifies breaking up parameters lists once they exceed ~80 characters.

```suggestion
  const ListAll({
    super.key,
    required this.listOfTasks,
    required this.title,
  });
```

Reply:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-a-trailing-comma-for-arguments-parameters-and-list-items-but-only-if-they-each-have-their-own-line